ironic/templates/conductor/conductor.yaml.j2

```diff
@@ -0,0 +1,118 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+spec:
+  replicas: {{ deployment.replicas }}
+  template:
+    metadata:
+      labels:
+        app: ironic-conductor
+    spec:
+      nodeSelector:
+        {{ deployment.control_node_label }}: enabled
+      # TODO(DTadrzak): it must be removed in future
+      securityContext:
+        runAsUser: 0
+      hostNetwork: true
+      hostPID: true
+      dnsPolicy: ClusterFirst
+      containers:
+        - name: ironic-conductor
+          image: {{ deployment.image.conductor }}
+          imagePullPolicy: Always
+          # TODO(DTadrzak): it must be removed in future
+          securityContext:
+            privileged: true
+          env:
+            - name: INTERFACE_NAME
+              value: "eth0"
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: COMMAND
+              value: "ironic-conductor"
+            - name: DEPENDENCY_JOBS
+              value: "ironic-db-sync"
+            - name: DEPENDENCY_SERVICE
+              value: "rabbitmq"
+            - name: DEPENDENCY_CONFIG
+              value: "/etc/ironic/ironic.conf,/etc/resolv.conf"
```

Comment:
same thing for /etc/resolv.conf we should use it as here: https://github.com/stackanetes/stackanetes/pull/99